tools/check/check.go

```diff
@@ -66,8 +66,8 @@ var (
 		("A path to a serialized check.Config protobuf. Any individual field flags will" +
 			" overwrite the message's associated field. By default, the file will be unmarshalled as binary," +
 			" but if it ends in .textproto, it will be unmarshalled as prototext instead."))
-	quiet   = flag.Bool("quiet", false, "If true, writes nothing the stdout or stderr. Success is exit code 0, failure exit code 1.")
-	verbose = flag.Int("verbosity", 0, "The output verbosity. Higher number means more verbose output")
+	quiet     = flag.Bool("quiet", false, "If true, writes nothing the stdout or stderr. Success is exit code 0, failure exit code 1.")
+	verbosity = flag.Int("verbosity", 0, "The output verbosity. Higher number means more verbose output")
```

Comment:
Correct argument name.

Reply:
Please update the readme. I see in attest we have verbose and verbosity whereas here we have quiet and verbosity. Just for consistency it might be better to add verbose over here also. It can be used to set the quiet variable so rest of the logic will be governed by a single variable.

Reply:
Verbose and quiet have different utilities. One is to stop output of even FATAL logs to stdout or stderr, whereas verbose is used to include INFO and WARN logs to stdout. I think we should not have a single variable to govern both.